test/RelayHubPenalizations.test.ts

```diff
@@ -469,44 +475,33 @@ contract('RelayHub Penalizations', function ([_, relayOwner, relayWorker, otherR
         gasPrice: relayCallArgs.gasPrice,
         to: relayHub.address,
         value: relayCallArgs.value,
-        // @ts-ignore
-        chainId: 1,
         data: encodedCall
-      })
+      }, transactionOptions)
 
       transaction.sign(Buffer.from(relayCallArgs.privateKey, 'hex'))
       return transaction
     }
 
     async function getDataAndSignatureFromHash (txHash: string, chainId: number): Promise<{ data: string, signature: string }> {
-      // @ts-ignore
-      const rpcTx = await web3.eth.getTransaction(txHash)
-      // eslint: this is stupid how many checks for 0 there are
-      // @ts-ignore
-      // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-      if (!chainId && parseInt(rpcTx.v, 16) > 28) {
+      const rpcTx: any = await web3.eth.getTransaction(txHash)
+      const vInteger = parseInt(rpcTx.v, 16)
```

Comment:
don't you need to add .replace(/0x/,'') ?

Reply:
Somehow it does work. This is a test though.